core/metacling/src/TClingCallbacks.cxx

```diff
@@ -316,10 +316,17 @@ bool TClingCallbacks::findInGlobalModuleIndex(DeclarationName Name, bool loadFir
    // FIXME: We should load only the first available and rely on other callbacks
    // such as RequireCompleteType and LookupUnqualified to load all.
    GlobalModuleIndex::FileNameHitSet FoundModules;
+   bool K = false;
 
    // Find the modules that reference the identifier.
    // Note that this only finds top-level modules.
-   if (Index->lookupIdentifier(Name.getAsString(), FoundModules)) {
+   if (Index->lookupIdentifier(Name.getAsString(), K, FoundModules)) {
+      if (K) {
+         if (gDebug > 2)
+            llvm::errs() << "\x1B[31m\"" << Name.getAsString() << "\" is a top level namespace,"
```

Comment:
Can we avoid adding console colors? Let's just use plain text which we can grep if the output is too much.